chrome/script.browserify.js

```diff
@@ -160,16 +180,73 @@ function getLoginData() {
   );
 }
 
+function loginToClipboard() {
+  var what = this.what;
+  chrome.runtime.sendNativeMessage(
+    app,
+    {action: "get", entry: this.entry},
+    function(response) {
+      if(chrome.runtime.lastError) {
+        error = chrome.runtime.lastError.message;
+        m.redraw();
+      }
+
+      chrome.tabs.query(
+        {lastFocusedWindow: true, active: true},
+        function(tabs) {
+          //do not copy login data if URL changed during search.
+          if(tabs[0].url == urlDuringSearch) {
```

Comment:
we probably don't care if URL changed, we have this check for filling a form to avoid filling credentials on a wrong website, but putting decrypted username/password to clipboard is an action that is safe to do regardless of the currently open tab.

Reply:
While I agree that it is not really needed, I still thing that we shouldn't write passwords to the clipboard when the site changed, so the user doesn't need to check check if it changed.

Reply:
I've just realized the following: we do similar url check in the background script, because background script is shared among all tabs, so there is a risk where you select credentials to be filled, but then quickly change tab in the browser. However this script is page-specific, so for example gpg is very slow and decryption takes seconds, if you change to a different tab during this time, the popup will close and the password will not be put in the clipboard. This can be reproducible by wrapping this whole `chrome.tabs.query` in a `setTimeout(..., 2000)` block.

Can you come up with an example when this check would still be useful? I'm fine with keeping it, but I want to know some examples of why we have it 🙂

Reply:
Ok, I didn't realize the script halts execution when one navigates. Yes you are right it's not needed.

Reply:
thanks, merging now 👍